terraform/cyhy_nmap_ec2.tf

```diff
@@ -35,7 +35,7 @@ resource "aws_instance" "cyhy_nmap" {
 
   root_block_device {
     volume_type = "gp2"
-    volume_size = "${local.production_workspace ? 100 : 8}"
+    volume_size = "${local.production_workspace ? 8 : 8}"
```

Comment:
Is 8GB what we had before?  Seems small.

Reply:
You're right- we used to have 20 GB here before (see [here](https://github.com/dhs-ncats/cyhy_amis/commit/b0b6b4175d080ced414494fe8691f03405b41207#diff-0ec35a026f754ebbbc04103a8e4c9031)).  I will change this back to 20 shortly.

Reply:
Resolved in [this commit](https://github.com/dhs-ncats/cyhy_amis/pull/146/commits/42a97111fa99339a441e52b207a0c0e465c6778a)